index.html

```diff
@@ -14,6 +14,77 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="main">
+
+<nav>
+  <a href="#">Resume</a>
+  <a href="#">Portfolio</a>
+  <a href="#">Contact</a>
+</nav>
+
+<div class="name">
+  
+  <img src="22851982_1944590418890091_5682228221849238783_n.jpg" alt="Me">
+  <h1>Laryna Billinghurst</h1>
+  <h2>To infinity and Beyond!</h2>
+  <line></line>
+  <p>Currently self-employed and a full-time student. I can work on projects, just give me an heads up!</p>
+  <button><a href="https://mail.google.com/mail/u/0/#inbox?compose=CllgCJNvvhtfntxCBWpCLhVjVlkkWThtWVNvFMNVkswSpsNnFCCPMWRpNWmWKJXlDMzJVvmQkWg" target="_blank">E-mail</a></button>
+ </div>
+
+ <div class="skills">
+   
+   <section>
+   <h1>My Initiative</h1>
+   <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque mollis purus at congue pharetra. In sapien lacus, porttitor et volutpat quis, luctus eu lorem. Duis in consectetur urna. Aliquam pellentesque vehicula risus, a condimentum velit feugiat ac. Proin mattis sapien vitae tempus luctus. Morbi porttitor viverra nunc, eget mollis nulla gravida ut. Morbi at feugiat ante. Integer tristique lectus eget sem euismod sodales. Integer mollis eu felis non porttitor. Sed at accumsan tortor. Fusce consectetur nibh in ex placerat egestas. Praesent a mattis mi, in mollis ex. Quisque vehicula nulla a sem blandit, eget blandit mauris maximus.
+    </p>
+    </section>
+
+    <section>
+      <h1>Skills</h1>
+    <p>Omni-Gamer</p>
+    <p>Coding/Programming</p>
+    <p>Drone Racing Pilot/Builder</p>
+    <p>Leadership</p>
+    <p>Sandbox Thinking</p>
+    </section>
+</div>
+
+<div class="work">
+  
+  <h1>Employment History</h1>
+  
+  <line class="line"></line>
+  
+  <table>
+   <tr>
+      <td>Years</td>
+      <td>Company</td>
+      <td>Title</td>
+      <td>Tech/Skill</td>
+   </tr>
+   <tr>
+      <td>5</td>
+      <td>Mario Party Wars</td>
+      <td>Executive Administrator</td>
+      <td>Leadership</td>
+   </tr>
+   <tr>
+     <td>7</td>
+     <td>Ice Pan</td>
+     <td>Assistant Manager</td>
+     <td>Management</td>
+   </tr>
+   <tr>
+     <td>2</td>
+     <td>Rideshare/Courier</td>
+     <td>Professional Driver</td>
+     <td>Self-Employment</td>
+   </tr>
+</table>
+</div>
+  
+
```

Comment:
Your HTML looks good I would suggest you be consistent with indenting for cleaner code.